styles/native/ts/core/variables.ts

```diff
@@ -157,11 +161,13 @@ button = merge(button, custom.button || {});
 let input: VariablesInput = {
     // Colors
     color: font.color,
+    colorDisabled: font.colorDisabled,
     errorColor: brand.danger,
     labelColor: font.color,
+    labelColorDisabled: font.color,
```

Comment:
Is this meant to be the default value of `font.color`, what about `font.colorDisabled`?

Reply:
I thought the designs said it was the same color as font.color, but after double checking it's actually slightly different. I added a new var `font.labelColorDisabled` for it